### ISSUE TITLE
No name is provided for "checkboxes", "clear" and "New Task" button: A11y_.NET 9 template_.NET 9 template_Project_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26128]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Name Role Value;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till checkboxes section.
4. Observe that No name is provided for "checkboxes", "clear" and "New Task" button.

### Actual Result:
No name is provided for "checkboxes", "clear" and "New Task" button.

### Expected Result:
Name should be provided for "checkboxes", "clear" and "New Task" button.

### User Impact:
Screen reader users will get the confuse if no name is provided for buttons available.

### Version with bug

9.0.10 SR1

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Relevant log output

```shell

```

**Comment by Vignesh-SF3580** [comment]
@AnKushSingh05 Reported issue was resolved in the following PRs.
https://github.com/dotnet/maui/pull/26920
https://github.com/dotnet/maui/pull/26919
https://github.com/dotnet/maui/pull/26903
https://github.com/dotnet/maui/pull/26921

**Comment by tj-devel709** [comment]
This one looks like it is just missing the clean up button
